PROTOCOL.md

```diff
@@ -965,6 +966,24 @@ class SetDirtyStateAction implements Action {
 ```
 </details>
 
+### 2.6.3. ExportSvgAction
+
+The client (or the server) sends a `ExportSvgAction` to indicate that diagram, which represents the current model state, should be exported in SVG format. The action provides the diagram svg as plain string.
```

Comment:
From this it is unclear to me who should handle the export SVG action in the end? Is there a clear responsibility between server or client? Also it seems that the action does not contain the file path so whoever handles it probably needs to derive the file name? Also since it is probably saved as a file (always?) it is probably something that can only be handled in an integration setting, is that right? Or how would that look like in the standalone usecase? How is overriding the file handled? Even if all of my questions are up to the implementation detail I believe it would be great to mention the individual aspects here.

Also there seems to be a typo: '... to indicate that THE diagram...'.